example/spec/helpers/granuleUtils.js

```diff
@@ -179,8 +179,8 @@ const waitForGranuleRecordInOrNotInList = async (stackName, granuleId, granuleIs
     return granuleIsIncluded ? ids.includes(granuleId) : !ids.includes(granuleId);
   },
   {
-    interval: 3000,
-    timeout: 240 * 1000,
+    interval: 10000,
+    timeout: 480 * 1000,
```

Comment:
Is there any context to these specific numbers? I'm a bit concerned that by just upping the timeouts, we may make the failures less frequent, but not have identified/resolved the cause of the issue

Reply:
The reconciliation report generation queries collection/granule records from ElasticSearch.  Many failures are due to the timeout when waiting granules to be inserted/deleted into ES.  In this case, wait for the granule added to the ES, then deleted from ES.
https://github.com/nasa/cumulus/blob/99ed6f1137d3191a5def4ddefa0fc93fbbda4820/example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js#L219-L222

Do you know if there is configuration to manage how soon the db records get propagated  into ES. @markdboyd

Reply:
The DB->ES syncing happens via a lambda triggered on updates to Dynamo, so no, there is no way to predict its timing that I know of. I assume updating these values has made the test more stable in local testing?

Reply:
Yeah, it should improve the stability.  We want to make sure the granules/collections are present/absent in ES before creating reconciliation reports.